<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_main</class>
 <widget class="QWidget" name="form_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>有道字典Python版</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QComboBox" name="cb_from_to"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_word">
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pb_translate">
     <property name="text">
      <string>翻译</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="chkB_detail">
     <property name="toolTip">
      <string extracomment="使用个人申请的APPKEY调有道收费查询接口"/>
     </property>
     <property name="text">
      <string>调收费接口</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextBrowser" name="tb_result">
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
